src/v2/Components/Artwork/ShelfArtwork.tsx

```diff
@@ -43,17 +44,24 @@ const ShelfArtwork: React.FC<ShelfArtworkProps> = ({
   const { containerProps, isSaveButtonVisible } = useSaveButton({
     isSaved: !!artwork.is_saved,
   })
+  const isHoverEffectEnabled = useFeatureFlag(
+    "force-enable-hover-effect-for-artwork-item"
+  )
+
+  const shouldShowHoverSaveButton =
+    isHoverEffectEnabled && (!!artwork.is_saved || isSaveButtonVisible)
 
   return (
-    <>
+    <div
+      {...containerProps}
+      data-test="artworkShelfArtwork"
+      data-testid="ShelfArtwork"
+    >
```

Comment:
Why do we need an extraneous wrapping `div`?

Reply:
We need to show the "save" button when the user hovered over the `ShelfArtwork` component, and not on the image (how it is now done on production). For this reason, we wrapped the contents of the component in a `div` and pass the `onMouseEnter` and `onMouseLeave` handlers

Reply:
Oh, I see — didn't realize the link didn't wrap the whole thing 👍